test/scripts/e2e_subs/rest-applications-endpoint.sh

```diff
@@ -31,5 +31,5 @@ call_and_verify "App parameter parsing error 1." "/v2/applications/-2" 400 "Inva
 call_and_verify "App parameter parsing error 2." "/v2/applications/not-a-number" 400 "Invalid format for parameter application-id"
 
 # Good request, but invalid query parameters
-call_and_verify "App invalid parameter" "/v2/applications/$APPID?this-should-fail=200" 400 'Unknown parameter detected: this-should-fail'
+call_and_verify "App invalid parameter" "/v2/applications/$APPID?this-should-not-fail=200" 200 '"global-state-schema":{"num-byte-slice":0,"num-uint":2}'
```

Comment:
does `/v2/applications/$APPID` and `/v2/assets/$ASSET_ID` return 200 for invalid params?

Reply:
The update removes this commit https://github.com/deepmap/oapi-codegen/commit/9cd29e22fd2f9b48236718bc7c53b4d7a91a4c02 from our fork. So the call will return the status that occurs when you execute the API, but ignore the query parameters that don't correspond to a valid parameter value.